sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePage.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.paging;
+
+import com.azure.core.util.IterableStream;
+
+/**
+ * Represents Page from service that has reference to next set of one or more pages,
+ * such a reference is known as continuation token.
+ *
+ * @param <C> Type of the continuation token
+ * @param <T> Type of the elements in the page
+ *
+ * @see ContinuablePagedFlux
+ */
+public interface ContinuablePage<C, T> {
+    /**
+     * @return an iterable stream of elements in the page.
```

Comment:
nit: add javadoc for method (it just has @return doc)

Reply:
Updated the java doc